.eslintrc.js

```diff
@@ -211,15 +211,6 @@ module.exports = {
       rules: {
         // does not work with wildcard imports. Mistakes will throw at runtime anyway
         'import/named': 'off',
-        'no-restricted-imports': [
```

Comment:
No need to do this anymore as package.json's `export` field controls visibility of exports.

Reply:
Which linting process is now controlling this? The typescript type checker?

Reply:
Node doesn't allow importing arbitrary files from the package when any subpath exports are defined.

Reply:
Ok, 👍 that assumes there is at least one Node.js process importing this natively